core/src/validator.rs

```diff
@@ -1872,4 +1882,8 @@ mod tests {
         };
         check_poh_speed(&genesis_config, Some(10_000));
     }
+
+    // bprumo TODO: need to write a test that brings up a validator from a snapshot, processes
+    // block store that crosses a full snapshot interval, and then ensure that
+    // AccountsBackgroundService gets the right value for last_full_snapshot_slot
```

Comment:
After thinking about this, I'm not sure how valuable it is. I already have tests for blockstore processor, and for AccountsBackgroundService. This would just be plumbing. Marking this as resolved.